app/containers/App/index.js

```diff
@@ -7,15 +7,28 @@
  */
 
 import React from 'react';
+import PropTypes from 'prop-types';
 import { Helmet } from 'react-helmet';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import ErrorBoundary from 'react-error-boundary';
 import styled from 'styled-components';
 import { Switch, Route } from 'react-router-dom';
 
+// Async bundles (code split-points)
 import HomePage from 'containers/HomePage/Loadable';
 import FeaturePage from 'containers/FeaturePage/Loadable';
 import NotFoundPage from 'containers/NotFoundPage/Loadable';
+
+// Dumb components
 import Header from 'components/Header';
 import Footer from 'components/Footer';
+import ErrorFallback from 'components/ErrorFallback';
```

Comment:
`react-error-boundary` contains `ErrorBoundaryFallbackComponent` should we consider reusing that?

Reply:
Thanks for pointing that out. I'll take a look.